modules/sbo-release-notes-1-3.adoc

```diff
@@ -0,0 +1,12 @@
+[id="sbo-release-notes-1-3_{context}"]
+// Module included in the following assembly:
+//
+// * applications/connecting_applications_to_services/sbo-release-notes.adoc
+:_content-type: REFERENCE
+= Release notes for {servicebinding-title} 1.3
+
+{servicebinding-title} 1.3 is now available on {product-title} 4.9, 4.10, and 4.11.
+
+[id="deprecation-notice-1-3_{context}"]
+== Removed functionality
+* In {servicebinding-title} 1.3, the Operator Lifecycle Manager (OLM) descriptor feature has been removed to improve resource utilization. As an alternative to OLM descriptors, you can use CRD annotations instead to declare binding data.
```

Comment:
Minimalism: Minor tweak but I think `instead` is redundant. 

`As an alternative to OLM descriptors, you can use CRD annotations to declare binding data.`

Reply:
Done